_zip/missingapi.yml

```diff
@@ -1,72 +1,42 @@
-  name: ToString
-  name.vb: ToString
-  fullName: XElement.ToString
-  fullName.vb: XElement.ToString
-- uid: System.Xml.Linq.XDocument.ToString*
-  href: https://msdn.microsoft.com/library/system.xml.linq.xdocument.tostring(v=vs.110).aspx
-  name: ToString
-  name.vb: ToString
-  fullName: XDocument.ToString
-  fullName.vb: XDocument.ToString
-- uid: System​.Runtime​.CompilerServices.IndexerNameAttribute
+- uid: System.Runtime.CompilerServices.IndexerNameAttribute
   href: https://msdn.microsoft.com/library/system.runtime.compilerservices.indexernameattribute(v=vs.110).aspx
   name: IndexerNameAttribute
   name.vb: IndexerNameAttribute
   fullName: System.Runtime.CompilerServices.IndexerNameAttribute
   fullName.vb: System.Runtime.CompilerServices.IndexerNameAttribute
-- uid: System.Reflection.MethodInfo.IsGenericMethodDefinition*
-  href: https://msdn.microsoft.com/library/system.reflection.methodinfo.isgenericmethoddefinition(v=vs.110).aspx
-  name: IsGenericMethodDefinition
-  name.vb: IsGenericMethodDefinition
-  fullName: MethodInfo.IsGenericMethodDefinition
-  fullName.vb: MethodInfo.IsGenericMethodDefinition
-- uid: System.Runtime.InteropServices.FieldOffsetAttribute
-  href: https://msdn.microsoft.com/library/system.runtime.interopservices.fieldoffsetattribute(v=vs.110).aspx
-  name: FieldOffsetAttribute
-  name.vb: FieldOffsetAttribute
-  fullName: System.Runtime.InteropServices.FieldOffsetAttribute
-  fullName.vb: System.Runtime.InteropServices.FieldOffsetAttribute
-- uid: System.Runtime.InteropServices.StructLayoutAttribute
-  href: https://msdn.microsoft.com/library/system.runtime.interopservices.structlayoutattribute(v=vs.110).aspx
-  name: StructLayoutAttribute
-  name.vb: StructLayoutAttribute
-  fullName: System.Runtime.InteropServices.StructLayoutAttribute
-  fullName.vb: System.Runtime.InteropServices.StructLayoutAttribute
-- uid: System.Runtime.InteropServices.DllImportAttribute
-  href: https://msdn.microsoft.com/library/system.runtime.interopservices.dllimportattribute(v=vs.110).aspx
-  name: DllImportAttribute
-  name.vb: DllImportAttribute
-  fullName: System.Runtime.InteropServices.DllImportAttribute
-  fullName.vb: System.Runtime.InteropServices.DllImportAttribute
-- uid: System.Runtime.InteropServices.OutAttribute
-  href: https://msdn.microsoft.com/library/system.runtime.interopservices.outattribute(v=vs.110).aspx
-  name: OutAttribute
-  name.vb: OutAttribute
-  fullName: System.Runtime.InteropServices.OutAttribute
-  fullName.vb: System.Runtime.InteropServices.OutAttribute
-- uid: System.Runtime.InteropServices.InAttribute
-  href: https://msdn.microsoft.com/library/system.runtime.interopservices.inattribute(v=vs.110).aspx
-  name: InAttribute
-  name.vb: InAttribute
-  fullName: System.Runtime.InteropServices.InAttribute
-  fullName.vb: System.Runtime.InteropServices.InAttribute
 - uid: System.Runtime.InteropServices.ComImportAttribute
```

Comment:
ComImportAttribute is also present in [API Browser](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.comimportattribute)